localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -280,6 +280,22 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if (
+      state_ptr_->count("nearest_voxel_transformation_likelihood") &&
+      std::stod((*state_ptr_)["nearest_voxel_transformation_likelihood"]) < 2.3) {
```

Comment:
Please use `converged_param_nearest_voxel_transformation_likelihood` declared in ndt_scan_matcher

Reply:
I will thank you.

Reply:
What I meant is to use this member variable instead of hard coding `2.3`.
https://github.com/autowarefoundation/autoware.universe/blob/23851b62dc9de76a0a2f61860bd2ace56948a07e/localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp#L140

Reply:
I understand now thank you.